src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportStatusBauService.cs

```diff
@@ -48,35 +51,62 @@ public class ImportStatusBauService : IImportStatusBauService
 
                     return queryResults
                         .OrderByDescending(result => result.Timestamp)
+                        .Where(result =>
+                        {
+                            var releaseId = new Guid(result.PartitionKey);
+                            var dataFileName = result.RowKey;
+                            var release =
+                                _contentDbContext.Releases
+                                    .Where(r => r.Id == releaseId)
+                                    .Include(r => r.Publication)
+                                    .FirstOrDefault();
+                            if (release != null)
+                            {
+                                _logger.LogWarning("No ContentDB Releases table row found!\n" +
+                                                   $"releaseId: \"{releaseId}\"");
+                            }
+
+                            var releaseFileRef = _contentDbContext.ReleaseFileReferences.FirstOrDefault(r =>
+                                r.ReleaseId == releaseId &&
+                                r.Filename == dataFileName &&
+                                r.ReleaseFileType == ReleaseFileTypes.Data);
+                            if (releaseFileRef != null)
+                            {
+                                _logger.LogWarning("No ContentDB ReleaseFileReferences table row found for data file!\n" +
```

Comment:
Wouldn't unnecessarily spread messages like this over multiple lines. This can be more succinctly written as:

> No ReleaseFileReference found with releaseId={releaseId}, dataFileName={dataFileName}

Note that specifying 'ContentDB' is unnecessary detail as there is only one ReleaseFileReference